index.bs

```diff
@@ -1757,8 +1756,8 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
     [=parse an optional 64-bit unsigned integer=] with |value|,
     "`source_event_id`", and 0.
 1. If |sourceEventId| is an error, return null.
-1. Let |expiry| be the result of running [=parse an expiry=] with |value|,
-    "`expiry`", and the [=max source expiry=].
+1. Let |expiry| be the result of running [=parse a duration=] with |value|,
```

Comment:
Let's extract the 1 day, 1 hour, and 1 hour clamp-start values into constants for ease of reference.